server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -113,114 +113,127 @@ class AddressIndexRepository @Inject()(
     * @param tokens tokens for the ES query
     * @return Search definition containing query to the ES
     */
-  def generateQueryAddressRequest(tokens: Seq[CrfTokenResult]): SearchDefinition = {
-
-    val tokensMap = Tokens.tokensToValidMap(tokens)
+  def generateQueryAddressRequest(tokens: Map[String, String]): SearchDefinition = {
 
     val query =
       bool {
```

Comment:
It is fine like it currently is (explicit).
It is soon to be rewritten completely, so it's pointless to try to refactor it currently.